artiq/coredevice/ad53xx.py

```diff
@@ -151,6 +157,9 @@ def __init__(self, dmgr, spi_device, ldac_device=None, clr_device=None,
         self.offset_dacs = offset_dacs
         self.core = dmgr.get(core)
 
+    # make driver accessible (see #1341)
+    voltage_to_mu = staticmethod(voltage_to_mu)
```

Comment:
`AD53xx.voltage_to_mu` should use `AD53xx.offset_dacs`

Reply:
Done. I can rebase, if you like.